etc/adminhtml/system.xml

```diff
@@ -50,68 +50,60 @@
             <field id="enableNewsletterSubscriptions">1</field>
           </depends>
         </field>
-        <field id="captchaType" translate="label" sortOrder="20" type="select" showInDefault="1" showInWebsite="0" showInStore="0">
+        <field id="captchaType" translate="label" sortOrder="20" type="select" showInDefault="1" showInWebsite="1" showInStore="0">
           <label>CAPTCHA Type</label>
           <source_model>Smaily\SmailyForMagento\Model\Config\Source\CaptchaType</source_model>
+          <comment>reCAPTCHA keys can be set under the Default Config.</comment>
           <depends>
             <field id="enableNewsletterSubscriptions">1</field>
             <field id="enableCaptcha">1</field>
           </depends>
         </field>
-        <field id="captchaApiKey" translate="label" type="text" sortOrder="30" showInDefault="1" showInWebsite="0" showInStore="0">
-          <label>Google API Key</label>
-          <comment><![CDATA[Create new site settings <a href="https://www.google.com/recaptcha/admin/create" target="_blank"> from here</a>.]]></comment>
-            <depends>
-              <field id="captchaType">google_captcha</field>
-              <field id="enableCaptcha">1</field>
-              <field id="enableNewsletterSubscriptions">1</field>
-          </depends>
-          <validate>required-entry</validate>
+      </group>
+      <group id="reCaptchaKeys" translate="label" type="text" sortOrder="30" showInDefault="1" showInWebsite="0" showInStore="0">
```

Comment:
For a separate group you would need to create an upgrade script that converts old reCAPTCHA configuration paths to new. We wouldn't want to break existing installations when users update the extension. Take a look at `Setup/UpgradeData.php` file.

Reply:
Added in commit.